lib/convertkit/utils.rb

```diff
@@ -0,0 +1,11 @@
+module ConvertKit
+  class Utils
+    class << self
+      def to_datetime(value)
+        return nil if !value.is_a?(String) || value.strip.empty?
+
+        DateTime.parse(value)
```

Comment:
Since we know and expect the time format to come back as ISO 8601 (e.g. "2023-07-06T19:31:31.000Z") which is what I see when using their API we should be specific here and say 

```suggestion
        DateTime.iso8601(value)
```

To ensure we don't accidentally parse something wrong here

Reply:
I agree. I'll make the change, but also verify with the ConvertKit team that this is the correct assumption.